src/explore-education-statistics-admin/src/components/ErrorBoundary.tsx

```diff
@@ -49,6 +43,11 @@ class ErrorBoundary extends React.Component<RouteComponentProps, State> {
         errorCode: undefined,
       });
     });
+
+    clients.forEach(client => {
```

Comment:
I see what you did there - nice :)  I wouldn't've thought of this way of linking a component to a non-react bit of js :)